ProfileManager.js

```diff
@@ -832,12 +838,17 @@ export default class ProfileManager {
     }
     const ephemeralSigner = await this._getAgentSigner(
       {profileAgentId: profileAgent.id, useEphemeralSigner: true});
+    let expires;
+    if(agentSigner && agentSigner.capability) {
+      expires = agentSigner.capability.expires;
+    }
```

Comment:
This is good, but even if there's no expiration on the agent signer capability, I still think we should put a default expiration here. So we should add an else that still sets a default expires.